fjord/base/tests/test_auth.py

```diff
@@ -44,27 +47,29 @@ def test_existing_user(self):
         new_user.backend = 'django_browserid.auth.BrowserIDBackend'
 
         # First, do it RAW!
-        get_request = RequestFactory().get(reverse('dashboard'))
-        get_request.user = new_user
-        get_request.session = self.client.session
+        post_request = RequestFactory().post(reverse('dashboard'))
+        post_request.user = new_user
+        post_request.session = self.client.session
 
         fv = FjordVerify()
         fv.user = new_user
-        fv.request = get_request
+        fv.request = post_request
 
         resp = fv.login_success()
-        eq_(302, resp.status_code)
-        eq_(resp.get('location'), '/')
+        eq_(200, resp.status_code)
+        body = json.loads(resp.content)
+        eq_(body['redirect'], '/')
 
         # Now do it with next!
-        get_request = RequestFactory().get(reverse('dashboard') + '?next=/foo')
-        get_request.user = new_user
-        get_request.session = self.client.session
+        post_request = RequestFactory().post(reverse('dashboard'), {'next': '/foo'})
```

Comment:
Oh, yeah. I changed it because `Verify` looks for `next` as a POST parameter. So this doesn't work the way it was originally tested.